huaweicloud/services/dcs/resource_huaweicloud_dcs_instance.go

```diff
@@ -948,25 +986,25 @@ func resizeDcsInstance(ctx context.Context, d *schema.ResourceData, meta interfa
 			return common.CheckDeleted(d, err, "DCS instance")
 		}
 		if instance.SpecCode != d.Get("flavor").(string) {
-			return fmtp.Errorf("[ERROR] Change flavor failed, "+
-				"after changed the DCS flavor still is: %s, expected: %s.", instance.SpecCode, specCode)
+			return fmt.Errorf("[ERROR] change flavor failed, "+
```

Comment:
`[ERROR]`, `[WARN]` and `[DEBUG]` are only available for log print methods.

Reply:
fixed